tests/scripts/thread-cert/Cert_9_2_07_DelayTimer.py

```diff
@@ -129,12 +133,99 @@ def test(self):
+
+        # Step 4: Leader MUST send a unicast MLE Child ID Response to the Router
+        _lpkts.filter_wpan_dst64(ROUTER).filter_mle_cmd(MLE_CHILD_ID_RESPONSE).must_next(
+        ).must_verify(lambda p: {ACTIVE_OPERATION_DATASET_TLV, ACTIVE_TIMESTAMP_TLV} < set(p.mle.tlv.type) and {
+            NM_CHANNEL_TLV, NM_CHANNEL_MASK_TLV, NM_EXTENDED_PAN_ID_TLV, NM_NETWORK_MASTER_KEY_TLV,
+            NM_NETWORK_MESH_LOCAL_PREFIX_TLV, NM_NETWORK_NAME_TLV, NM_PAN_ID_TLV, NM_PSKC_TLV, NM_SECURITY_POLICY_TLV
+        } <= set(p.thread_meshcop.tlv.type) and p.mle.tlv.active_tstamp == LEADER_ACTIVE_TIMESTAMP)
+
+        # Step 6: Leader automatically sends a MGMT_ACTIVE_SET.rsp to the Router
+        _lpkts.filter_ipv6_dst(ROUTER_RLOC).filter_coap_ack(MGMT_ACTIVE_SET_URI).must_next().must_verify(
+            lambda p: p.coap.code == 68 and p.thread_meshcop.tlv.state == 1)
+
+        # Step 7: Leader multicasts a MLE Data Response with the new information
+        _lpkts.filter_LLANMA().filter_mle_cmd(MLE_DATA_RESPONSE).must_next().must_verify(
+            lambda p: {SOURCE_ADDRESS_TLV, LEADER_DATA_TLV, ACTIVE_TIMESTAMP_TLV, NETWORK_DATA_TLV} <= set(
+                p.mle.tlv.type) and {NM_BORDER_AGENT_LOCATOR_TLV, NM_COMMISSIONER_SESSION_ID_TLV} <= set(
+                    p.thread_meshcop.tlv.type) and p.thread_nwd.tlv.stable == [0] and p.mle.tlv.active_tstamp ==
+            ROUTER_ACTIVE_TIMESTAMP)
+
+        # Step 10: Leader MUST send a unicast MLE Data Response to the Router
+        _lpkts.filter_wpan_dst64(ROUTER).filter_mle_cmd(MLE_DATA_RESPONSE).must_next(
+        ).must_verify(lambda p: {ACTIVE_OPERATION_DATASET_TLV, ACTIVE_TIMESTAMP_TLV} < set(p.mle.tlv.type) and {
+            NM_CHANNEL_TLV, NM_CHANNEL_MASK_TLV, NM_EXTENDED_PAN_ID_TLV, NM_NETWORK_MASTER_KEY_TLV,
+            NM_NETWORK_MESH_LOCAL_PREFIX_TLV, NM_NETWORK_NAME_TLV, NM_PAN_ID_TLV, NM_PSKC_TLV, NM_SECURITY_POLICY_TLV
+        } <= set(p.thread_meshcop.tlv.type) and p.mle.tlv.active_tstamp == ROUTER_ACTIVE_TIMESTAMP)
+
+        # Step 12: Leader sends a MGMT_PENDING_SET.rsp to the Router with Status = Accept
+        _lpkts_coap.filter_ipv6_dst(ROUTER_RLOC).filter_coap_ack(MGMT_PENDING_SET_URI).must_next().must_verify(
+            lambda p: p.coap.code == 68 and p.thread_meshcop.tlv.state == 1)
+
+        # Step 13: Leader sends a multicast MLE Data Response
+        _lpkts.filter_LLANMA().filter_mle_cmd(MLE_DATA_RESPONSE).must_next().must_verify(
+            lambda p: {
+                SOURCE_ADDRESS_TLV, LEADER_DATA_TLV, ACTIVE_TIMESTAMP_TLV, PENDING_TIMESTAMP_TLV, NETWORK_DATA_TLV
+            } <= set(p.mle.tlv.type) and p.thread_nwd.tlv.stable == [0] and p.mle.tlv.active_tstamp ==
+            ROUTER_ACTIVE_TIMESTAMP and p.mle.tlv.pending_tstamp == ROUTER_PENDING_TIMESTAMP)
+
+        # Step 14: The DUT MUST send MGMT_DATASET_CHANGED.ntf to the Router
+        _lpkts_coap.filter(lambda p: p.wpan.dst16 == ROUTER_RLOC16).filter_coap_request(
```

Comment:
```suggestion
        _lpkts_coap.filter_wpan_dst16(ROUTER_RLOC16).filter_coap_request(
```
add `filter_wpan_dst16` ?

Reply:
added